src/EA.Weee.Web.Tests.Unit/Areas/Admin/ViewModels/EditRegisteredCompanyOrganisationDetailsViewModelTests.cs

```diff
@@ -51,13 +51,13 @@ public void GivenModel_CompaniesRegistrationNumberShouldHaveStringLengthAttribut
                 .BeDecoratedWith<StringLengthAttribute>(a => a.MaximumLength.Equals(EnvironmentAgencyMaxFieldLengths.CompanyRegistrationNumber)
                                                              && a.MinimumLength.Equals(7)
                                                              && a.ErrorMessage.Equals(
-                                                                 "The company registration number should be 7 to 15 characters long"));
+                                                                 "The Company registration number should be 7 to 15 characters long"));
```

Comment:
@stevelee-civica  this one is ok as it is in a sentence not at the start of a sentence.